lib/data/external/utils.js

```diff
@@ -0,0 +1,8 @@
+const utils = {
+    logHelper(log, level, description, error, dataStoreName) {
```

Comment:
Being nit-picky, I was rather thinking of a local helper that would embed the `error from AWS` message and specifics, so that it has the same format everywhere. I don't think this one has much value especially since you can't log specific objects like the normal log function. But if you don't have more time to spend on this it's alright, the essential is that the error is logged ;)